src/Api/Controller/CreateDiscussionController.php

```diff
@@ -65,7 +65,7 @@ protected function data(ServerRequestInterface $request, Document $document)
         $actor = $request->getAttribute('actor');
         $ipAddress = array_get($request->getServerParams(), 'REMOTE_ADDR', '127.0.0.1');
 
-        if (! $request->getAttribute('bypassFloodgate')) {
+        if ($actor->can('floodgate.throttling') && ! $request->getAttribute('bypassFloodgate')) {
```

Comment:
I would turn this around as well - throttling should be on for everyone (the current default) unless explicitly disabled.

That also takes us to naming... what about something like `postWithoutThrottle`? That's the thing you are *permitted* to do.